db/schema.rb

```diff
@@ -10,8 +12,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2020_02_27_214321) do
-
+ActiveRecord::Schema.define(version: 2020_05_12_171425) do
   # These are extensions that must be enabled in order to support this database
```

Comment:
Is the schema always updated when new migrations are created? Or when is the version updated?

Reply:
Yes, the schema is always updated when new migrations are created. The database, however, is not changed until you db:migrate those changes

Reply:
Schema's are updated when you run the command "rails db:migrate"

It looks at the migration file, and then updates the schema accordingly.